tests/test_course_meta_loader.py

```diff
@@ -0,0 +1,122 @@
+from raise_data.loaders import course_meta_loader
+import boto3
+import pytest
+import botocore.stub
+import csv
+from io import StringIO, BytesIO
+from sqlalchemy.dialects.postgresql import insert
+from raise_data.dashboard.schema import Course
+
+
+@pytest.fixture(autouse=True)
+def setup_database_table():
+    """This fixture clears and populates the Course table before each test
+    run in this file
+    """
+    with course_meta_loader.session_factory.begin() as session:
+        session.query(Course).delete()
+        insert_stmt = insert(Course).values(
+            [
+                {'id': 1, 'name': 'Name 1', 'term': 'Term 1', 'district': ''},
+                {'id': 2, 'name': 'Name 2', 'term': 'Term 2', 'district': ''}
+            ]
+        )
+        session.execute(insert_stmt)
+
+
+def test_course_meta_loader(mocker):
+
+    s3_client = boto3.client("s3")
+    s3_stubber = botocore.stub.Stubber(s3_client)
+
+    mock_dict_writer_data_1 = [
+        {
+            "course_id": 1,
+            "district": "Halo School District",
+        },
+        {
```

Comment:
Oh -- Another thought as far as test fixture: Let's add a row here that doesn't exist in the database just to confirm nothing goes :boom: . That could potentially happen depending on the relative sequence / ordering of some of the automation.